src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
+
+  @include media($nav-width) {
+    border-bottom: 1px solid $color-gray-light;
+  }
+
+  a {
+    border-bottom: none;
+  }
+
+  em {
+    display: block;
+    font-size: $h5-font-size;
+    font-style: normal;
+    font-weight: $font-bold;
+    margin: 0;
+
+    @include media($nav-width) {
+      display: block;
+      font-size: 2.4rem;
+      line-height: $heading-line-height;
+    }
+  }
+
+  .usa-search {
+    margin-bottom: 1.5rem;
+
+    @include media($nav-width) {
+      float: right;
+      margin-bottom: 0;
+      max-width: 21.5rem;
+    }
+  }
+}
+
+.usa-navbar {
```

Comment:
**tl;dr** there’s a lot going on with the headers. I recommend that we revisit this idea with validated user research. Implementing this is not something we shouldn’t do (I even have a couple of ideas), but only that it’s not as easy as it seems right now.

Now for the longer bit...

When we talk about the nav, it’s important to understand the foundation of the component and code. All navs start out the same: they’re essentially constructed as a mobile slide out nav and everything within it. This mobile nav then gets transformed into a horizontal oriented nav at large screen sizes, either the inset basic header or the extended header dropped below. It’s a single unit that functions together.

Although the markup is distinct, the functionality (i.e. responsive interaction via Sass) is interwoven. All these distinct pieces are orchestrated together in the stylesheets. The responsive behavior happens in the same code block, selector by selector.

See lines [6-9](https://github.com/18F/web-design-standards/blob/1a552233e5b017cb3a803217a34972ac89671bb9/src/stylesheets/components/_header.scss#L6...#L9), you’ll see three z-index variables for the header. Keeping them together ensures this is easy to manage and people know what is doing what and can find it in the same place. Splitting them apart would be confusing to manage. We wouldn’t put them in variables file bc they’re not global variables but scoped to the header.

>This might help in creating the ability to using each of these sets of styles are their own. For example, we could possibly run into issues when want to use nav specific styles in another component, but are required to either remove or incorporate header styles.

Before drawing to conclusions, I’d want to ask @ericadeahl. Has our user research validated this need? Do we have evidence of people wanting to use the navigation in another component that's not the header?

Using a header with a nav, and then a nav again somewhere else on the page will cause conflict bc the code is a mobile nav and that would overlap with each other.

On desktop, We use a custom variable `$nav-width`, if you’re using the header and then you want to use the nav for another use case, you can’t change this variable to something else. They will be the same, but you don’t want them to be the same, you’d want to be able to control the width for your use case. The current code doesn’t support that. You’d have to re-copy this code and then add a new variable, you can see this variable is used many times throughout the code.

@juliaelman @msecret